var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -148,6 +171,15 @@ class Gcc(AutotoolsPackage):
     # See https://gcc.gnu.org/gcc-5/changes.html
     conflicts('languages=jit', when='@:4')
 
+    # NVPTX offloading supported by limited languages
```

Comment:
we might want to add that the `nvptx` variant only works with GCC 8.2.0 or newer?

Reply:
OpenMP offloading with nvptx has been available since gcc 7, OpenAcc even before that I think (gcc 5?).

Reply:
I added a restriction to require 7 or above. We could push potentially push that down to 5 with some openacc testing, but I haven't tried that at all (only openmp)